stripe_official.php

```diff
@@ -511,51 +513,51 @@ public function install()
+                || !Configuration::updateValue(self::ENABLE_ALIPAY, 0, false, $shopGroupId, $shopId)
+                || !Configuration::updateValue(self::ENABLE_OXXO, 0, false, $shopGroupId, $shopId)) {
+                return false;
+            }
+
+            if (!$this->installOrderState()) {
+                return false;
+            }
+
+            return true;
         } catch (PrestaShopDatabaseException $e) {
-            ProcessLoggerHandler::logError(
+            Stripe_officialClasslib\Extensions\ProcessLogger\ProcessLoggerHandler::logError(
                 $e->getMessage(),
                 null,
                 null,
                 'Stripe_official - install'
             );
-            ProcessLoggerHandler::closeLogger();
+            Stripe_officialClasslib\Extensions\ProcessLogger\ProcessLoggerHandler::closeLogger();
             return false;
         } catch (PrestaShopException $e) {
-            ProcessLoggerHandler::logError(
+            Stripe_officialClasslib\Extensions\ProcessLogger\ProcessLoggerHandler::logError(
                 $e->getMessage(),
                 null,
                 null,
                 'Stripe_official - install'
             );
-            ProcessLoggerHandler::closeLogger();
-            return false;
-        }
-
-        $shopGroupId = Stripe_official::getShopGroupIdContext();
-        $shopId = Stripe_official::getShopIdContext();
-
-        // preset default values
-        if (!Configuration::updateValue(self::MODE, 1, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::REFUND_MODE, 1, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::MINIMUM_AMOUNT_3DS, 50, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_IDEAL, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_SOFORT, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_GIROPAY, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_BANCONTACT, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_FPX, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_EPS, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_P24, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_SEPA, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_ALIPAY, 0, false, $shopGroupId, $shopId)
-            || !Configuration::updateValue(self::ENABLE_OXXO, 0, false, $shopGroupId, $shopId)) {
-                 return false;
-        }
-
-        if (!$this->installOrderState()) {
+            Stripe_officialClasslib\Extensions\ProcessLogger\ProcessLoggerHandler::closeLogger();
             return false;
         }
-
-        return true;
```

Comment:
Are you sure to remove return true at then end of install ?

Reply:
i have place the return true statement in the try block (line 541)